package.json

```diff
@@ -3,8 +3,14 @@
   "version": "0.1.0",
   "description": "An express server to expose VROOM as a web API.",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node src/index.js"
+    "test": "npm run test:other && npm run test:js",
+    "start": "node src/index.js",
+    "fix:other": "npm run prettier --write",
+    "fix:js": "npm run test:js --fix",
+    "fix": "npm run fix:js && npm run fix:other",
+    "prettier": "prettier \"**/*.{json,md,scss,yaml,yml}\"",
+    "test:other": "npm run prettier --list-different",
```

Comment:
Running `npm run test:other` is outputting all matching files. I'd expect to see only suggested changes so it looks like the `--list-different` option is ignored.

Reply:
Works now, was using the wrong npm syntax.. thought it worked like yarn - dough! ;-)